### PR TITLE
cdc: add description about ticdc lightning compatibility

### PR DESCRIPTION
**Pull request by 3AceShowHand**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.5 (TiDB 8.5 versions)
- [x] v8.4 (TiDB 8.4 versions)
- [x] v8.3 (TiDB 8.3 versions)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-compatibility.md [hunk 1]
```diff
@@ -7,10 +7,25 @@ summary: 了解 TiCDC 兼容性相关限制和问题处理。
...
+逻辑模式以向 TiDB 执行 SQL 语句的方式导入数据。这部分数据可以正常被 TiCDC 同步。操作步骤如下：
+
+1. 创建 changefeed
+2. 启动 Lightning，以逻辑模式模式导入数据
+
+物理模式以向 TiKV 插入 SST 文件的方式导入数据。当前阶段，TiCDC 和 Lightning 物理模式并不兼容，不同步由该模式导入的数据。
+
+如果下游集群是 TiDB，操作步骤如下：
+
+1. 分别向上下游 TiDB 集群导入数据，保证上下游集群的数据一致性
```
Comment (flowbehappy) [inline ticdc/ticdc-compatibility.md:25]:
```suggestion
1. 分别向上下游 TiDB 集群用 Lightning 导入数据，保证上下游集群的数据一致性
```

File: ticdc/ticdc-compatibility.md [hunk 2]
```diff
@@ -7,10 +7,25 @@ summary: 了解 TiCDC 兼容性相关限制和问题处理。
...
+2. 创建 changefeed，同步上游集群正常写入的数据到下游
+
+如果下游是其他类型数据系统，可以自行编写程序，将 Lightning 的输入文件内容写入到下游系统中，再创建 changefeed 同步正常写入的数据。
```
Comment (flowbehappy) [inline ticdc/ticdc-compatibility.md:28]:
```suggestion
如果下游是非 TiDB 系统，需要先将 Lightning 的输入文件内容通过对应系统的离线导入工具导入到下游系统中，再创建 changefeed 以同步 SQL 写入的增量数据。
```

File: ticdc/ticdc-compatibility.md [hunk 3]
```diff
@@ -7,10 +7,25 @@ summary: 了解 TiCDC 兼容性相关限制和问题处理。
...
+2. 启动 Lightning，以逻辑模式模式导入数据
+
+物理导入模式以向 TiKV 插入 SST 文件的方式导入数据。当前阶段，TiCDC 和 Lightning 物理模式并不兼容，不同步由该模式导入的数据。
+
+如果下游集群是 TiDB，操作步骤如下：
+
+1. 分别向上下游 TiDB 集群用 Lightning 导入数据，保证上下游集群的数据一致性
+2. 创建 changefeed，同步上游集群后续通过 SQL 写入的数据到下游
+
+如果下游是非 TiDB 系统，需要先将 Lightning 的输入文件内容通过对应系统的离线导入工具导入到下游系统中，再创建 changefeed 以同步 SQL 写入的增量数据。
```
Comment (Oreoxmt) [inline ticdc/ticdc-compatibility.md:29]: Please remove the following lines (L30-L34)
https://github.com/pingcap/docs-cn/blob/b93b7b682f1b5605152689ff3c80a5dcc6e63b1c/ticdc/ticdc-compatibility.md?plain=1#L30-L34

File: tidb-lightning/tidb-lightning-compatibility-and-scenarios.md [hunk 4]
```diff
@@ -19,6 +19,8 @@ summary: 了解 IMPORT INTO 和 TiDB Lightning 与日志备份和 TiCDC 的兼
 
 - `IMPORT INTO` 与日志备份以及 TiCDC 均不兼容。原因是 `IMPORT INTO` 的导入过程也是将源数据编码后的 KV Pairs 直接 Ingest 到 TiKV。
 
+Lightning 和 TiCDC 共同工作时的操作步骤，参考 [TiCDC 和 Lightning 的兼容性](/ticdc/ticdc-compatibility.md#ticdc-和-lightning-的兼容性)
```
Comment (Oreoxmt) [inline tidb-lightning/tidb-lightning-compatibility-and-scenarios.md:22]:
```suggestion
- 如果你需要在同一个集群上同时使用 TiDB Lightning 和 TiCDC，请参考 [TiCDC 与 TiDB Lightning 的兼容性](/ticdc/ticdc-compatibility.md#ticdc-与-tidb-lightning-的兼容性)。
```

Please move this note about TiDB Lightning and TiCDC compatibility to line 19, as it should be grouped with the other TiDB Lightning-related content in lines 16-18.

File: ticdc/ticdc-compatibility.md [hunk 5]
```diff
@@ -7,10 +7,25 @@ summary: 了解 TiCDC 兼容性相关限制和问题处理。
 
 本文介绍了与 TiCDC 有关的一系列兼容性问题及其处理方案。
 
-<!--
-## 组件兼容性矩阵
+## TiCDC 和其他 TiDB 组件的兼容性
 
-TODO
+### TiCDC 和 Lightning 的兼容性
```
Comment (Oreoxmt) [inline ticdc/ticdc-compatibility.md:12]:
```suggestion
## TiCDC 与 TiDB Lightning 的兼容性
```

File: ticdc/ticdc-compatibility.md [hunk 6]
```diff
@@ -7,10 +7,25 @@ summary: 了解 TiCDC 兼容性相关限制和问题处理。
 
 本文介绍了与 TiCDC 有关的一系列兼容性问题及其处理方案。
 
-<!--
-## 组件兼容性矩阵
+## TiCDC 和其他 TiDB 组件的兼容性
```
Comment (Oreoxmt) [inline ticdc/ticdc-compatibility.md:10]:
```suggestion
```

File: ticdc/ticdc-compatibility.md [hunk 7]
```diff
@@ -7,10 +7,25 @@ summary: 了解 TiCDC 兼容性相关限制和问题处理。
...
 
-TODO
+### TiCDC 和 Lightning 的兼容性
+
+Lightning 支持[逻辑导入模式](/tidb-lightning/tidb-lightning-logical-import-mode.md)和[物理导入模式](/tidb-lightning/tidb-lightning-physical-import-mode.md)。
```
Comment (Oreoxmt) [inline ticdc/ticdc-compatibility.md:14]:
```suggestion
[TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 支持[逻辑导入模式](/tidb-lightning/tidb-lightning-logical-import-mode.md)和[物理导入模式](/tidb-lightning/tidb-lightning-physical-import-mode.md)两种数据导入模式。本章节介绍这两种模式与 TiCDC 的兼容性，以及同时使用 TiDB Lightning 和 TiCDC 时的操作步骤。
```

File: ticdc/ticdc-compatibility.md [hunk 8]
```diff
@@ -7,10 +7,25 @@ summary: 了解 TiCDC 兼容性相关限制和问题处理。
...
+
+逻辑导入模式以向 TiDB 执行 SQL 语句的方式导入数据。这部分数据可以正常被 TiCDC 同步。操作步骤如下：
+
+1. 创建 changefeed
+2. 启动 Lightning，以逻辑模式模式导入数据
```
Comment (Oreoxmt) [inline ticdc/ticdc-compatibility.md:19]:
```suggestion
在逻辑导入模式下，TiDB Lightning 通过执行 SQL 语句导入数据。此模式与 TiCDC 兼容。你可以按照以下步骤同时使用 TiDB Lightning 逻辑导入模式和 TiCDC 进行数据同步：

1. 创建 changefeed，详情参考[创建同步任务](/ticdc/ticdc-manage-changefeed.md#创建同步任务)。
2. 启动 TiDB Lightning 并使用逻辑模式模式导入数据，详情参考[使用逻辑导入模式](/tidb-lightning/tidb-lightning-logical-import-mode-usage.md)。
```

File: ticdc/ticdc-compatibility.md [hunk 9]
```diff
@@ -7,10 +7,25 @@ summary: 了解 TiCDC 兼容性相关限制和问题处理。
...
+
+物理导入模式以向 TiKV 插入 SST 文件的方式导入数据。当前阶段，TiCDC 和 Lightning 物理模式并不兼容，不同步由该模式导入的数据。
+
+如果下游集群是 TiDB，操作步骤如下：
+
+1. 分别向上下游 TiDB 集群用 Lightning 导入数据，保证上下游集群的数据一致性
+2. 创建 changefeed，同步上游集群后续通过 SQL 写入的数据到下游
+
+如果下游是非 TiDB 系统，需要先将 Lightning 的输入文件内容通过对应系统的离线导入工具导入到下游系统中，再创建 changefeed 以同步 SQL 写入的增量数据。
```
Comment (Oreoxmt) [inline ticdc/ticdc-compatibility.md:28]:
```suggestion
在物理导入模式下，TiDB Lightning 通过向 TiKV 插入 SST 文件的方式导入数据。TiCDC 与此模式不兼容，不支持同步通过物理模式导入的数据。如果你需要同时使用 TiDB Lightning 物理导入模式和 TiCDC，可以根据 TiCDC 下游系统的类型选择以下解决方案：

- 下游系统是 TiDB 集群：

    1. 使用 TiDB Lightning 分别向上下游 TiDB 集群导入数据，以确保两个集群的数据一致性。
    2. 创建 changefeed，用于同步后续通过 SQL 写入的增量数据。详情参考[创建同步任务](/ticdc/ticdc-manage-changefeed.md#创建同步任务)。

- 下游系统不是 TiDB 集群：

    1. 使用下游系统提供的离线导入工具，将 TiDB Lightning 的输入文件导入到下游系统。
    2. 创建 changefeed，用于同步后续通过 SQL 写入的增量数据。详情参考[创建同步任务](/ticdc/ticdc-manage-changefeed.md#创建同步任务)。
```

File: ticdc/ticdc-compatibility.md (hunk 6 above)
Comment (Oreoxmt) [reply ticdc/ticdc-compatibility.md:10]: Simplify the heading structure by removing the redundant parent heading, as there is only one compatibility section specifically for TiDB Lightning.

**Review by Oreoxmt — APPROVED** [review]

**Review by flowbehappy — APPROVED** [review]

File: ticdc/ticdc-compatibility.md [hunk 10]
```diff
@@ -7,15 +7,24 @@ summary: 了解 TiCDC 兼容性相关限制和问题处理。
 
 本文介绍了与 TiCDC 有关的一系列兼容性问题及其处理方案。
 
-<!--
-## 组件兼容性矩阵
+## TiCDC 和 TiDB Lightning 的兼容性
```
Comment (Oreoxmt) [inline ticdc/ticdc-compatibility.md:10]:
```suggestion
## TiCDC 与 TiDB Lightning 的兼容性
```

**Pull request merged**